lib/tasks/multitenant_db_merge.rake

```diff
@@ -0,0 +1,344 @@
+      id_mapping[obj_for_migrate[i]['id'].to_i] = x[0]
+    end
+
+    # Spot check
+    puts "Spot check:"
+    puts "Raw row: #{obj_for_migrate.first}"
+    puts "Clean row: #{clean_rows[0]}"
+    puts "Inserted record: #{model.find(id_mapping[obj_for_migrate.first['id']]).attributes}\n\n"
+
+    id_mapping
+  end
+
+  # Port configs
+  config_map = -> (x) {
+    x.except('id', 'fund_id', 'config_value', 'created_at', 'updated_at')
+     .merge({
+       'fund_id' => @fund_id,
+       'created_at' => x['created_at'].asctime.in_time_zone("America/New_York"),
+       'updated_at' => x['updated_at'].asctime.in_time_zone("America/New_York"),
+       'config_value' => JSON.parse(x['config_value'])
+     })
+  }
+  @config_mappings = easy_mass_insert Config, 'configs', config_map
+
+  # Port clinics
+  clinic_map = -> (x) {
+    x.except('id', 'fund_id', 'created_at', 'updated_at')
+     .merge({
+       'fund_id' => @fund_id,
+       'created_at' => x['created_at'].asctime.in_time_zone("America/New_York"),
+       'updated_at' => x['updated_at'].asctime.in_time_zone("America/New_York")
+     })
+  }
+  @clinic_mappings = easy_mass_insert Clinic, 'clinics', clinic_map
+
+  # Port lines
+  line_map = -> (x) {
+    x.except('id', 'fund_id', 'created_at', 'updated_at')
+     .merge({
+       'fund_id' => @fund_id,
+       'created_at' => x['created_at'].asctime.in_time_zone("America/New_York"),
+       'updated_at' => x['updated_at'].asctime.in_time_zone("America/New_York")
+     })
+  }
+  @line_mappings = easy_mass_insert Line, 'lines', line_map
+
+  # Port users
+  user_map = -> (x) {
+    x.except('id', 'fund_id', 'line', 'created_at', 'updated_at')
+     .merge({
+       'fund_id' => @fund_id,
+       'created_at' => x['created_at'].asctime.in_time_zone("America/New_York"),
+       'updated_at' => x['updated_at'].asctime.in_time_zone("America/New_York")
+     })
+  }
+  @user_mappings = easy_mass_insert User, 'users', user_map
+
+  # Port patients
+  patient_map = -> (x) {
+    x.except('id', 'fund_id', 'created_at', 'updated_at', 'line_id', 'clinic_id', 'pledge_generated_by_id', 'pledge_sent_by_id', 'last_edited_by_id')
+     .merge({
+       'fund_id' => @fund_id,
+       'created_at' => x['created_at'].asctime.in_time_zone("America/New_York"),
+       'updated_at' => x['updated_at'].asctime.in_time_zone("America/New_York"),
+       'clinic_id' => @clinic_mappings[x['clinic_id']],
+       'line_id' => @line_mappings[x['line_id']],
+       'pledge_generated_by_id' => @user_mappings[x['pledge_generated_by_id']],
+       'pledge_sent_by_id' => @user_mappings[x['pledge_sent_by_id']],
+       'last_edited_by_id' => @user_mappings[x['last_edited_by_id']]
+     })
+  }
+  @patient_mappings = easy_mass_insert Patient, 'patients', patient_map
+
+  # Port archived patients
+  archived_patient_map = -> (x) {
+    x.except('id', 'fund_id', 'created_at', 'updated_at', 'line_id', 'clinic_id', 'pledge_generated_by_id', 'pledge_sent_by_id', 'last_edited_by_id')
```

Comment:
This has `last_edited_by_id` excepted, but not included back in in the merge below?

Reply:
winds up being a noop -- `last_edited_by_id` isn't on archived patient, turns out. I'll take it out of here regardless.